grails-app/jobs/mama/ng/scheduler/MessageJob.groovy

```diff
@@ -26,7 +28,13 @@ class MessageJob {
         messages.each { Message message ->
             def schedule = message.schedule
 
-            def success = httpRequestService.postText(schedule.endpoint)
+            def body = [
+                "schedule-id": schedule.id,
+                "message-id": message.id,
+                "send-counter": schedule.sendCounter
```

Comment:
When send counter changes, the old message will be deleted. Only the most current messages associated with the schedule will exist.